### DISCUSSION TITLE
Blank tabs

### DISCUSSION THREAD

**Discussion by djsimmz** [Micke-K/IntuneManagement#166] [General]
Has anything changed on the new version?

Can't seem to get any of current policies to display, all tabs just blank.
![Capture](https://github.com/Micke-K/IntuneManagement/assets/145697703/916bffdc-add1-4dec-b84b-70eb1755d7f6)

**Comment by djsimmz** [comment]
Never mind, TLS issue.

Anyone else has the problem in powershell just run:

[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12  

Then run Start-IntuneManagement.ps1 in same window.

**Reply by PukinDogs** [reply]
Thanks for this dj, totaly worked for my use. Now I just to edit the start.cmd to reflect this.

For my own knowledge, I had some trouble identifying what was going wrong, how did you establish that it was a TLS error?

**Comment by Micke-K** [comment]
Hello,

This is a bit strange because TLS is set in the Initialize-CloudAPIManagement function in CloudAPIPowerShellManagement.psm1. This is the only thing function that Start-IntuneManagement.ps1 is calling.

<img width="736" height="65" alt="image" src="https://github.com/user-attachments/assets/b06d914e-149a-433c-9e96-6abdf925462c" />

PowerShell 5.1 or 7?

Cheers!
